proto/src/determined/api/v1/trial.proto

```diff
@@ -29,6 +29,9 @@ message SummarizedMetric {
   // A possibly down-sampled series of metric readings through the progress of
   // the trial in wall time.
   repeated DataPointTime time = 4;
+  // A possibly down-sampled series of metric readings through the progress of
+  // the trial in epoch.
+  repeated DataPointEpoch epochs = 5;
```

Comment:
my preference would be to go ahead and make it so that we just have `DataPoint` with optional fields for `time` and `epochs` (in addition to `batches` and `value`).

in theory, with different x-axes, we could get different downsamplings, but it's probably sufficient to just downsample according to the default x-axis (batches). whatever sampling efficiency we get by downsampling separately is likely not worth the added complexity and computation IMO

Reply:
I agree with this and will make the refactor